File: src/test/java/com/coors/PersonTest.java
package com.coors;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class PersonTest {
    @Test
    public void bmiTest() {
        Person person = new Person();
        person.setWeight(66.5f);
        person.setHeight(1.7f);

        // 左邊為 期待值 ， 右邊為實際
        Assertions.assertEquals(66.5f/(1.7f * 1.7f) , person.bmi());
    }
}

File: src/test/kotlin/com/coors/PersonTest.kt
package com.coors.kotlin

import com.coors.Human
import com.coors.Person
import org.junit.Test
import org.junit.jupiter.api.Assertions

class PersonTest {
    @Test
    fun humanBmiTest() {
        val human = Human("coors",66.5f, 1.7f)
        Assertions.assertEquals((66.5f/(1.7f*1.7f)) , human.bmi())
    }

    @Test
    fun personBmiTest() {
        val person = Person()
        person.weight = 66.5f
        person.height = 1.7f

        Assertions.assertEquals((66.5f/(1.7f*1.7f)) , person.bmi())
    }
}